<?xml version="1.0"?>
<robot 
    xmlns:xacro="http://www.ros.org/wiki/xacro" name="test" >

  <xacro:include filename="$(find aerial_robot_model)/robots/test/test_common.xacro" />
  <xacro:include filename="$(find aerial_robot_model)/robots/test/test_link.urdf.xacro" />

  <!--  <xacro:test_link parent="link0" self="link1"/> -->
  <xacro:test_link />

</robot>
